pkgs/desktops/gnome-3/misc/gnome-autoar/default.nix

```diff
@@ -42,7 +42,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     platforms = platforms.linux;
     maintainers = teams.gnome.members;
-    license = licenses.lgpl21;
+    license = licenses.lgpl21Only;
```

Comment:
Looks like lgpl21Plus to me

https://gitlab.gnome.org/GNOME/gnome-autoar/-/blob/0.3.0/gnome-autoar/gnome-autoar.h#L9

Reply:
Hum, you are right.